velox/core/QueryConfig.h

```diff
@@ -219,6 +222,13 @@ class QueryConfig {
     return get<int32_t>(kTestingSpillPct, 0);
   }
 
+  /// Returns the start partition bit which is used with 'kSpillPartitionBits'
+  /// together to calculate the spilling partition number.
+  uint8_t spillStartPartitionBit() const {
```

Comment:
This is a nice improvement, but I don't see where the new property is being used. Would you point me to the call site?

Also, it would be easier to review if this change was introduced in a separate PR.

Reply:
Make sense. I leave it a separate PR to actually use it.